scripts/changelog/publiccode_updater.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This is an updater for the utility "standard-version" that increase the versionName value
+ * for publiccode.yml file.
+ * Replace the line:
+ * softwareVersion: $VERSION
+ * with the new generated version.
+ *
+ * and the line:
+ * releaseDate: '$DATE'
+ * with the today date.
+ *
+ */
+
+const softwareVersionRegex = /(softwareVersion: )(.+)/gm;
+const releaseDateRegex = /(releaseDate: ')(.+)(')/gm;
+
+module.exports.readVersion = function (contents) {
+  // return the 2nd group of the regex (the version)
+  return softwareVersionRegex.exec(contents)[2];
+};
+
+function replaceReleaseDate(match, version, p1, p2, p3) {
```

Comment:
```suggestion
function replaceReleaseDate(version, p1, p2, p3) {
```
Maybe it could be removed since it is unused?
If yes the same happens in `replaceVersionName`

Reply:
Changed variable name in 68c863178aa5236a194a19866b4ed9529b5e2dc6